Documentation/policy/language.rst

```diff
@@ -34,7 +34,18 @@ Labels Based
 ------------
 
 Label-based L3 policy is used to establish policy between endpoints inside the
-cluster managed by Cilium.
+cluster managed by Cilium. Label-based L3 policies are defined by using an
+`EndpointSelector` inside a rule to choose what kind of traffic that can be
+received (on ingress), or sent (on egress). An empty `EndpointSelector` allows
+all traffic. The examples below demonstrate this in further detail.
+
+When Cilium is running with kubernetes as an orchestrator, the network policies
+are typically scoped by ``Namespace``. If an explicit namespace selector is not
+defined in the rule's `EndpointSelector`, then an implicit selector on the
+policy's namespace will be added to the rule's `EndpointSelector`. A special
```

Comment:
`to the rule's EndpointSelector.` to `to the rule's EndpointSelector, which will be the same namespace where the policy is installed`.